sys/crypto/modes/cbc.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     sys_crypto_modes
+ * @ingroup     sys_crypto_mode_cbc
```

Comment:
If you just use `sys_crypto` here, the introduction of the new groups below would not be required.

Reply:
You are right, fixed now.